src/apps/dfx/dfx.definition.ts

```diff
@@ -0,0 +1,30 @@
+import { Register } from '~app-toolkit/decorators';
+import { AppDefinition } from '~app/app.definition';
+import { GroupType, ProtocolAction } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const DFX_DEFINITION = {
+  id: 'dfx',
+  name: 'dfx',
+  description: 'DFX.Finance is a decentralized foreign exchange protocol optimized for stablecoins',
+  url: 'https://app.dfx.finance/',
+  groups: {
+    curve: { id: 'dfx-curve', type: GroupType.TOKEN },
```

Comment:
Try to keep the key `curve` as the camelcase of the ID `dfx-curve`, i.e:

```ts
dfxCurve: { id: 'dfx-curve', type: GroupType.TOKEN },
```

Reply:
Fix https://github.com/Zapper-fi/studio/pull/347/commits/9a0983e387d58d2c6e1c6e7ed9c40dc9b8880323